.travis.yml

```diff
@@ -44,6 +45,18 @@ before_install:
         printf '\ny\n\ny\ny\n' | ./google-cloud-sdk/install.sh &&
         cd $TRAVIS_BUILD_DIR;
   fi
+- gcloud components update -q
```

Comment:
Before this line, maybe try:

```yaml
  gcloud config set --scope=user disable_updater false
```

Totally just guessing here, though, based on the [Travis error](https://travis-ci.org/GoogleCloudPlatform/nodejs-docs-samples/jobs/89557546#L244):

> ```bash
> $ gcloud components update -q
> You cannot perform this action because the component manager has been disabled for this installation.
> ```

Then I googled the error and got the line in source: http://123.108.47.253/google-cloud-sdk/.install/.backup/lib/googlecloudsdk/core/config.py which lead me to that `disable_updater` option.